src/tests/multihost/ipa/test_adtrust.py

```diff
@@ -379,3 +381,88 @@ def test_nss_get_by_name_with_private_group(self, multihost):
         assert cmd_adm.returncode == 0, 'Something wrong with setup!'
         assert cmd_usr.returncode == 0, \
             f"pysss_nss_idmap.getsidbyname for {username} failed"
+
+    @staticmethod
+    def test_idview_override_group_custom(multihost, create_aduser_group):
+        """
+        :title: IPA clients fail to resolve override group names in custom view
+        :id: 7a0dc871-fdad-4c07-9d07-a092baa83178
+        :bugzilla:
+          https://bugzilla.redhat.com/show_bug.cgi?id=2004406
+          https://bugzilla.redhat.com/show_bug.cgi?id=2031729
+        :description: Overriding both user and group names and ids in
+          an idview for user and group from AD results in error in sssd
+          when running id command.
+        :setup:
+         1. Create user and group (group1) on AD.
+         2. Make AD user member of group1.
+         2. Create additional group (group2) on AD.
+        :steps:
+         1. ID views to override AD groupname and gid of group1.
+         2. ID views to override AD groupname and gid of group2.
+         3. ID view to override AD username, uid and gid (to gid of group2).
+         4. Run an "id" command for the override user.
+         client
+        :expectedresults:
+          1. View with an override is created.
+          2. View with an override is created.
+          3. User override is added to the view.
+          4. Id command succeeds, group override is visible, all groups are
+          properly resolved.
```

Comment:
Indent 'properly' to start from under 'Id'

Reply:
Done.